### PR TITLE
Index page markup

### PR DESCRIPTION
**Pull request by brew**
This PR partly fulfils the 'index page' requirement for #785. 

- Add sorting by Rank and Place by clicking table headers (default sort by Place)
- Add dataset openness indicators to table cells for each place/dataset
- Add popover with openness summary with icons and action buttons (Read, Update, Review). Summary derives from new `QuestionShort` property.
- Add 'Add' and 'Review' actions to cells without an entry

Note, the index page is currently missing button styles https://github.com/okfn/opendatacensus/issues/785#issuecomment-257824739

### CODE REVIEW

File: census/views/includes/dataviews/table_places.html [hunk 1]
```diff
@@ -1,88 +1,114 @@
 {% import "macros/popovers.html" as popovers %}
 
 <div class="table-responsive">
-  <div class="sexyTableContainer">
+  <div class="sexyTableContainer" style="margin: 0px 0px 20px;">
```
Comment (nightsh) [inline census/views/includes/dataviews/table_places.html]: Can these inline styles be extracted into a stylesheet?

File: census/views/includes/dataviews/table_places.html [hunk 2]
```diff
@@ -1,88 +1,114 @@
...
   <div class="sexyDragContainer kinetic-active kinetic-moving-left kinetic-moving-up" style="cursor: move;">
-  <table id="places_overview_table" class="table data-table sexyTable">
+  <table id="places_overview_table" class="table data-table sexyTable" style="margin: 0px;">
```
Comment (nightsh) [inline census/views/includes/dataviews/table_places.html]: Inline style.

File: census/views/includes/dataviews/table_places.html [hunk 3]
```diff
@@ -1,88 +1,114 @@
...
     <thead>
       <tr>
-        <th class="sort_rank header headerSortDown" title="{{ gettext('Sort by Rank') }}">{{ gettext('Rank') }}</th>
-        <th class="sort_place header" title="{{ gettext('Sort by A-Z') }}">{{ gettext('Place') }}</th>
+        <th class="sort_rank header" title="{{ gettext('Sort by Rank') }}" style="width: 71px;">{{ gettext('Rank') }}</th>
```
Comment (nightsh) [inline census/views/includes/dataviews/table_places.html]: Inline style.

File: census/views/includes/dataviews/table_places.html [hunk 4]
```diff
@@ -1,88 +1,114 @@
...
+        <th class="sort_place header headerSortDown" title="{{ gettext('Sort by A-Z') }}" style="width: 71px;">{{ gettext('Place') }}</th>
```
Comment (nightsh) [inline census/views/includes/dataviews/table_places.html]: Inline style.

File: census/views/includes/dataviews/table_places.html [hunk 5]
```diff
@@ -1,88 +1,114 @@
...
         {% for dataset in datasets %}
-          <th style="width: 71px;">
-            <div><span class="dataset-title-context" data-toggle="popover" title="{{ dataset.name }}" data-content="{{ dataset.description|e }}">{{ dataset.name }}</span></div>
+          <th>
+            <div><span class="dataset-title-context" data-toggle="popover" title="{{ dataset.name }}" data-content="{{ dataset.description|e }}" data-original-title="{{dataset.name}}">{{ dataset.name }}</span></div>
           </th>
         {% endfor %}
-        <th class="header" title="{{ gettext('Sort by Score') }}">{{ gettext('Score') }}</th>
+        <th class="header" style="width: 72px;">{{ gettext('Score') }}</th>
```
Comment (nightsh) [inline census/views/includes/dataviews/table_places.html]: Inline style.

**Comment by brew** [comment]
Thanks - agree, ideally they wouldn't be inline.

**Pull request merged**